README.md

```diff
@@ -20,22 +29,117 @@ These take the form of:
 - RESTful API
 - Tableau integration to perform data querying and dashboard creation
 
-## Technologies
+## Project Roadmap
 
-blah
+![Screenshot_Excalidraw](https://user-images.githubusercontent.com/115073814/217223449-26e6a315-d01c-4a11-a743-ceee6c4e60d7.png)
 
-## Setup
+### Summary of above roadmap:
 
-blah
+1. Kafka Consumer in EC2 takes logs from Kafka Data stream every second
+2. Parses through logs and cleans data, passes into SQLite table ready for querying for Live Dashboard, displays information on current ride and recent rides
+3. At any point cleaned data suggests current user exceeds heart rate limit during ride, email alert trigger sent using Amazon simple email service (SES)
+4. For a given ride for a specific user, at the end of their current ride, two csv files (user_info & ride_info) sent to s3 bucket (file storage service)
+5. AWS Lambda function reads csv files from s3 bucket and extracts key metrics (using Pandas) for long term storage in AWS RDS postgres table
+6. RDS comprised of two tables, users and rides, joined by user_id
+7. Tableau dashboard connected to RDS schemas and visuals created from it, live streaming the information
+8. API hosted on EC2 performing SQL queries from RDS
+9. Daily report hosted on AWS Lambda, extracting key metrics and formatted as an insightful summary, ran daily using Crontab
 
-## Credits
+## Usage
 
-Direct contributors to the repository:
+### Live Dashboard
+
+1. You will find the live dashboard up and running here: [Live Dashboard](http://18.130.141.140:8080/)
+2. Top left button allows user to switch between viewing the current ride (user details along with their current performance) and recent ride details.
+
+#### Current Ride:
+<img width="1411" alt="Screenshot_LiveDashboard" src="https://user-images.githubusercontent.com/115073814/217261046-79872c89-567d-4f6e-bf45-7550f1fbcbdf.png">
+
+#### Recent Rides:
+<img width="1414" alt="Screenshot_LiveDashboardRecent" src="https://user-images.githubusercontent.com/115073814/217261372-560e20da-d1ef-4a44-8d4f-6825283be560.png">
+
+### RESTful API
+
+[API](https://jzrx9wfk25.execute-api.eu-west-2.amazonaws.com/), can search the following endpoints:
+
+```GET``` ```/ride/:id```
+Get a ride with a specific ID
+
+```GET``` ```/rider/:user_id```
+Get rider information (e.g. name, gender, age, avg. heart rate, number of rides)
+
+```GET``` ```/rider/:user_id/rides```
+Get all rides for a rider with a specific ID
+
+```DELETE``` ```/ride/:id```
+Delete a with a specific ID
+
+```GET``` ```/daily```
+Get all of the rides in the current day
+
+```GET``` ```/daily?date=01-01-2020```
+Get all rides for a specific date
```

Comment:
nice addition of all the API endpoints

Reply:
I definitely didn't just nick that from the curriculum